utils/i18n.go

```diff
@@ -80,12 +80,12 @@ func GetTranslationsBySystemLocale() (i18n.TranslateFunc, error) {
 	}
```

Comment:
There is one more error at L57: `fmt.Errorf(fmt.Sprintf("Unable to find i18n directory at %q", dir))`. Let's change that too, and directly use `fmt.Errorf`.

Reply:
fixed